### ISSUE TITLE
Build error with 

### ISSUE THREAD

**Issue by atgreen** [shamazmazum/cl-forward-diff#2]
This commit... https://github.com/shamazmazum/cl-forward-diff/commit/d66ce3fd4d2df5e861ce4544cbbf7da913e9d721

```
; caught ERROR:
;   READ error during COMPILE-FILE:
;   
;     Lock on package SB-C violated when interning *COMPILATION-UNIT* while in
;     package CL-FORWARD-DIFF.
```

https://github.com/ocicl/cl-forward-diff/actions/runs/11231598146/job/31221366229

**Comment by shamazmazum** [comment]
Thanks for reporting! It must be fixed in 7804948.

**Comment by atgreen** [comment]
Indeed it is...
```
$ ocicl list cl-forward-diff
cl-forward-diff(ghcr.io/ocicl):
 latest
 20241008-7804948
 20241003-0d6b3fa
 20240728-9ce122a
```
